modules/nw-network-config.adoc

```diff
@@ -25,6 +25,11 @@ Phase 1:: After entering the `openshift-install create install-config` command.
 +
 For more information on these fields, refer to "Installation configuration parameters".
 
+[NOTE]
+====
+Set `networking.machineNetwork` to match the CIDR which the preferred NIC resides.
```

Comment:
To me, this sentence doesn't entirely make sense grammatically.

But the larger issue is that, I don't believe this is necessarily the best or only placement for this. This also needs to be reflected in the `networking.machineNetwork` field description itself:

`modules/installation-configuration-parameters.adoc`

But the explanation is missing some context, I think. How is a NIC preferred, exactly? And why does the CIDR align with this?

At least for the cluster network, the default route on the node seems to govern what the preferred/selected NIC is, but I don't know if etcd uses the cluster network or not.

In any case, I think this needs a deeper explanation, but I can't divine exactly what is absent.